src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -116,7 +138,7 @@
           DependsOnTargets="DiscoverTestInputs;CheckTestCategories"
           AfterTargets="CheckTestCategories"
           Inputs="@(RunTestsForProjectInputs)"
-          Outputs="$(TestsSuccessfulSemaphore);$(TestPath)%(TestTargetFramework.Folder)/$(XunitResultsFileName);$(CoverageOutputFilePath)"
+          Outputs="$(TestsSuccessfulSemaphore);$(TestPath)%(TestNugetTargetMoniker.Folder)/$(XunitResultsFileName);$(CoverageOutputFilePath)"
```

Comment:
Do we expect that we will ever have more then on TestNugetTargetMoniker set for a given build/test run?